clients/admin-ui/src/features/system/form.ts

```diff
@@ -10,6 +10,7 @@ export interface FormValues
     progress?: DataProtectionImpactAssessment["progress"];
     link?: DataProtectionImpactAssessment["link"];
   };
+  customFieldValues?: Record<string, string | string[]>;
```

Comment:
nit. Does it makes sense to make a type that something like

```typescript
type CustomFieldValues = Record<string, string | string[]>;
```

Then it could be reused in the few spots where this type signature is used.

Reply:
hehe yeah the thought crossed my mind but I got lazy 😅 will do!